docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,148 @@
+
+Battery Historian tool also allows an A/B comparison of two bug reports, highlighting the key differences in the battery-related metrics.
+
+> [!NOTE]
+> To use the Battery Historian feature, ensure that the Smart Development Bridge (SDB) is enabled to issue commands to Tizen devices. 
+
+## Access Battery Historian
+
+To access and open the Battery Historian tool, use one of the following:
+
+ - In the Tizen Studio menu, select **Tools \> Battery Historian.**
+ - In the Tizen Studio toolbar, click **Battery Historian**  ![baticon](./media/batt1.png).
+ - In Tizen Studio main window, press the combination of the **Alt+Shift+B** keys. 
+
+The **Battery Historian** window appears. 
+
+![](./media/bug1.png)
+
+The Battery Historian tool window comprises of the following components:
+
+ - **Browse**: This component allows you to browse and select a bug report file.
+ - **Switch to Bug Report Comparison**: This component allows you to compare and analyze two different bug report files.
+
+> [!NOTE]
+> The Battery Historian tool only supports **.txt** or **.zip** format for bug report file.
+
+## Use Battery Historian Tool
+
+The Battery Historian tool essentially requires a bug report file to process the battery consumption stats of a Tizen device. This bug report file contains all the relevant data regarding the battery consumption. You can generate the bug report using the SDB that dumps the battery data to your development machine and creates a report.
+
+The Battery Historian tool takes the bug report file as an input to analyze and compare the stats. After processing the report data, the Battery Historian uses HTML format to visualize the stats in a web browser.
+
+To use the Battery Historian tool, ensure that you have the bug report files readily available. If you do not have the bug report file with, you can generate one using the following procedure:
+
+### Generate Bug Report File
+
+To generate a bug report file, follow these steps:
+
+1. From your Linux Terminal, enter the following commands:
+
+    ```
+
+    $] sdb root on
+
+    $] sdb shell
+
+    ```
+
+2. From your Tizen device terminal, enter the following commands:
+
+    ```
+
+    sh-3.2] dumpsys batterstats \&gt; /tmp/battery\_dump.txt
+
+    sh-3.2] exit
+
+    ```
+
+3. From Linux Terminal, enter the following commands::
+
+    ```
+
+    $] sdb pull /tmp/battery\_dump.txt
+
+    ```
+
+After you execute the commands, the `batterystats.txt` file is generated in the specified directory using the optional path argument.
+
+> [!NOTE]
+> If no path is specified, the file gets created in your home directory.
+
+### View and Analyze Battery Data
+
+To view and analyze your data, follow these steps:
+
+1. In Battery Historian Main Window, click **Browse**.
+
+2. Select the appropriate bug report file.
+
+3. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in the file. After the analysis is done, a battery stats chart appears:
```

Comment:
Would you please check the "battery stats chart" in this sentence?
According to the flow it seems to be "battery historian chart".